backends/cuda-reg/ceed-cuda-reg-basis.c

```diff
@@ -31,12 +31,14 @@ typedef CeedScalar real;
 //------------------------------------------------------------------------------
 // Read non interleaved dofs
 //------------------------------------------------------------------------------
+/*
```

Comment:
I highlighted code that is not currently used (some from my changes, some from older changes). We should decide what to do with these functions. Untested/unused code gets stale fast. I'd recommend deleting.

Reply:
I'm okay with deleting.

Reply:
done